conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+
+@pytest.fixture
+def set_up(page):
+    page.goto("https://apex.heits.digital/#")
```

Comment:
NIT: here's a good example where we could use an enum in the future. We could store specific environment data (base_url and other vars) and use an env variable for communicating the targeted environment